h1st/core/trust/lime_model_explainer.py

```diff
@@ -3,27 +3,26 @@
 
 class LIMEModelExplainer:
     def __init__(self, decision, model, data):
-        self.model = model
-        self.data = data
-        self.feature_names = list(self.data["train_df"].columns)
+        self._model = model
+        self._data = data
+        self.feature_names = list(self._data["train_df"].columns)
         self.decision_input = decision[0]
-        self.lime_explainer()
-        self.explain_decision()
-        ## Only for testing if plots are generated
-        # self.plot_lime_explanation()
+        self._lime_explainer()
+        self._explain_decision()
+        # self._plot_lime_explanation()
 
-    def plot_lime_explanation(self):
+    def _plot_lime_explanation(self):
         self.explanation.show_in_notebook(show_table=True, show_all=False)
```

Comment:
if you need to expose attributes "explanation" and "explainer" externally, consider them as getter/setter. Otherwise, make it as private with _ @timroz24